class.json-api-endpoints.php

```diff
@@ -267,11 +267,32 @@ function input( $return_default_values = true, $cast_and_filter = true ) {
 			break;
 		}
 
-		if ( !$cast_and_filter ) {
-			return $return;
+		if ( isset( $this->api->query['force'] ) 
+		    && 'secure' === $this->api->query['force']
+		    && isset( $return['secure_key'] ) ) {
+			$this->api->post_body = $this->get_secure_body( $return['secure_key'] );
```

Comment:
It would be really cool to have a comment here since later when we forget how this works it will be hard to follow what is going on